inc/healthcheck/namespace.php

```diff
@@ -10,7 +10,8 @@ function bootstrap() {
 		require_once __DIR__ . '/class-cli-command.php';
 		WP_CLI::add_command( 'healthcheck', __NAMESPACE__ . '\\CLI_Command' );
 	}
-	if ( ! isset( $_SERVER['REQUEST_URI'] ) || '/healthcheck/' !== $_SERVER['REQUEST_URI'] ) {
+
+	if ( ! isset( $_SERVER['REQUEST_URI'] ) || strpos( $_SERVER['REQUEST_URI'], '__healthcheck' ) == false ) {
```

Comment:
This should check that it's at the start of the URL, not just in it, otherwise we might hijack too many URLs. Also, should use a strict check (`===`).
```suggestion
	if ( ! isset( $_SERVER['REQUEST_URI'] ) || strpos( $_SERVER['REQUEST_URI'], '/__healthcheck' ) === 0 ) {
```

Reply:
Hmm, ```Healthcheck can be accessed via /__healthcheck or /__healthcheck/ with any query string``` - I thought it should be accessible everywhere?

Reply:
Only at the root (e.g. `humanmade.com/__healthcheck`), but any query string should be allowed (e.g. `humanmade.com/__healthcheck?whatever`)

Reply:
I am not sure why I understood it otherwise. Done.